yapapi/executor/__init__.py

```diff
@@ -539,10 +539,10 @@ async def worker_starter() -> None:
                     len(agreements_to_pay),
                 )
                 await asyncio.wait(
-                    {process_invoices_job}, timeout=10, return_when=asyncio.ALL_COMPLETED
+                    {process_invoices_job}, timeout=20, return_when=asyncio.ALL_COMPLETED
```

Comment:
It has been suggested that the requestor should wait a bit longer for the invoices.